examples/plot_anomaly_comparison-3D_SKlearn-Copy2.py

```diff
@@ -0,0 +1,293 @@
+"""
+============================================================================
+Comparing anomaly detection algorithms for outlier detection on 3D toy datasets
+============================================================================
+
+
+Brief overview
+---------------------------------
+This example is an extension of anomaly detection comparison from ``sckit-learn 2.7.1``. These experiment applied six algorithms on different 3D toy dataset. Four algorithms are in sklearn, which are ``sklearn.covariance.EllipticEnvelope``, ``sklearn.svm.OneClassSVM``, ``sklearn.ensemble.IsolationForest``, and ``sklearn.neighbors.LocalOutlierFactor``. The other two algorithms are not from sklearn and are needed to install separately. They are ``Extended Isolation Forest`` and ``USPORF``. The outlier detecting performance is measure using. Accuracy score and ROC graph from ``sklearn.metrics.roc_auc_score``.
+
+
+Installation 
+---------------------------------
+Beside ``sklearn``, ``matplotlib``, and ``numpy``, you need to install these packages::
+
+    pip install eif
+    pip install rerf
+
+
+Data simulation 
+---------------------------------
+Here are the dafault parameters of the data set, which can be change according to the user:
+
+  * ``d_noise = 10`` - the number of dimensional noise (uniform noise)
+  * ``n_samples = 500`` - the number of sample
+  * ``outlliers_fraction = 0.15`` - ratio of outliers to all sample
+  
+The data has three information dimension, which is followed figure 1 in the paper:
+    
+  * Madhyastha, Meghana, et al. "Geodesic Learning via Unsupervised Decision Forests." arXiv preprint arXiv:1907.02844 (2019).
+
+There are five dataset:
+  
+  * Linear line
+  * Spiral helix
+  * Sphere shell
+  * Three aligned gaussian mixture
+  * Three misaligned gaussian mixture
+  
+  
+Performance measurement 
+---------------------------------
+Each algorithm will predict the outliers and inliers. There are two performance measurement in this example.
+    
+  * ``sklearn.metrics.accuracy_score``
+  * ``sklearn.metrics.roc_auc_score``
+      * ``sklearn.metrics.roc_curve``
+
+Result and disscussion 
+---------------------------------
+``sklearn.covariance.EllipticEnvelope`` shows best result in high ``d_noise = 10``. However, since robust covariance create a ellptical envelope for inliers, we need more test on a misaligned data
```

Comment:
all lines should be <88 chars

Reply:
I will make a change accordingly in the most recent submission: [commits 81ba3f4](https://github.com/NeuroDataDesign/scikit-learn/pull/9/commits/81ba3f4408fc3f025ecedf7aad8aa4e28458317d)